src/App.tsx

```diff
@@ -2,108 +2,42 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { PreparedPost } from './types/PreparedPost';
+
+import { PostList } from './components/PostList';
+
+function getUserById(userId: number): User | undefined {
+  return usersFromServer
+    .find(author => author.id === userId) || undefined;
```

Comment:
find by default return undefined if nothing found
```suggestion
  return usersFromServer.find(author => author.id === userId);
```